packages/core/core/src/BundleGraph.js

```diff
@@ -825,19 +827,19 @@ export default class BundleGraph {
   }
 
   getSiblingBundles(bundle: Bundle): Array<Bundle> {
```

Comment:
Should that function return a set instead?
But if every callsite does `.map` or filter anyway it's more convenient as it is...

Reply:
Yeah I had the same thought and came to roughly the same conclusion 😅